### DISCUSSION TITLE
What happens to LFS data after changing plans?

### DISCUSSION THREAD

**Discussion by muHashh** [community/community#141588] [Enterprise]
### Select Topic Area

Question

### Body

Hi,

I currently have a plan that provides extra storage on LFS which I am using. However, I am looking to remove that plan, and was wondering what happens to the data once the plan is removed (the data that's more than the 1GB allowed on the free tier plan)? Does it get deleted, or am I able to access and move it somewhere else down the line?

**Comment by queenofcorgis** [comment]
@muHashh thank you for your patience! 

We do not delete the LFS data. If you downgrade or stop paying for the LFS plan you would just lose access. 

If the limits increase under metered billing, or if you pays for your LFS data, you will regain access.

The only circumstances where we delete LFS data is if the repo has been deleted, or if a user removes an object, and requests support to purge the data. Check out this [further reading](https://docs.github.com/en/repositories/working-with-files/managing-large-files/removing-files-from-git-large-file-storage#git-lfs-objects-in-your-repository).
